cmd/grpcurl/grpcurl.go

```diff
@@ -566,10 +566,11 @@ func prettify(docString string) string {
 	// from each line in the doc string
 	j := 0
 	for _, part := range parts {
+		part = strings.TrimSpace(part)
```

Comment:
This is the first actual change. This is in reply to a comment you made on a commit from #56.